backend/src/main/java/de/neuefische/backend/security/config/SecurityConfig.java

```diff
@@ -48,7 +52,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .antMatchers("/").permitAll()
                 .antMatchers("/api/**").authenticated()
                 .antMatchers("/auth/login").permitAll()
-                .and().addFilterBefore(jwtAuthFilter, UsernamePasswordAuthenticationFilter.class)
+                .and()
+                .addFilterBefore(jwtAuthFilter, UsernamePasswordAuthenticationFilter.class)
```

Comment:
Hier würde ich .addFilterBefore(...) in die Zeile 55 einrücken: also 
55  .and().addFilterBefore(...)